### DISCUSSION TITLE
Case für openDTU Fusion

### DISCUSSION THREAD

**Discussion by philippsandhaus** [tbnobody/OpenDTU#1269] [Q&A] [answered]
Hallo zusammen,

ich habe mir das sehr schön designte openDTU Fusion Board und war davon ausgegangen, dass es irgendwo schon ein 3D-druckbares Modell für ein Case geben. Nun finde ich hierzu leider nichts, auch nicht auf den bekannten Plattformen thingiverse oder printables. Übersehe ich etwas oder gibt es wirklich kein frei verfügbares Modell? Wenn nein, würde ich wahrscheinlich versuchen selber eines zu designen.

**Comment by Solarteur** [comment, marked as answer]
Vielleicht dieses hier?
https://3printd-solution.eu/Fusion-Gehaeuse

**Reply by mastertape** [reply]
Er fragte nach einer frei verfügbaren STL Datei zum selber drucken - nicht zum selber kaufen ;)

**Reply by philippsandhaus** [reply]
Alles gut, unter dem Link gibt es auch die passende STL- Datei zum Download. Danke noch einmal. Ich merke gerade, dass ich damals gar nicht geantwortet hatte. 👍

**Reply by mastertape** [reply]
Oops jupp nach genauen suchen gut versteckt ist da auch die STL Datei....

Hiermal ein Link damit man es direkt findet:
https://3printd-solution.eu/Fusion-Gehaeuse#tab-Sonstiges

Ansonsten noch der Tipp das man in der Beschreibung auf "Sonstiges " drücken muss dahinter versteckt sich dann der Download link
